resources/views/components/server/sidebar.blade.php

```diff
@@ -10,9 +10,11 @@
         href="{{ route('server.private-key', ['server_uuid' => $server->uuid]) }}">Private Key
     </a>
     @if ($server->isFunctional())
-        <a class="menu-item {{ $activeMenu === 'cloudflare-tunnels' ? 'menu-item-active' : '' }}"
-            href="{{ route('server.cloudflare-tunnels', ['server_uuid' => $server->uuid]) }}">Cloudflare
-            Tunnels</a>
+        @if (!$server->isLocalhost())
```

Comment:
Following the Cloudflare Tunnels doc, but what is the correct way now to connect to a secondary server routed through cloudfared?

Reply:
Do I use the [cloudflared app](https://coolify.io/docs/services/cloudflared)?

Reply:
I temporarily used an IP from a Tailscale overlay network to add and validate the server.  Then I used Cloudflare Tunnels option to connect the second server permanently.